js/apps/admin-ui/src/clients/add/CapabilityConfig.tsx

```diff
@@ -26,13 +36,34 @@ export const CapabilityConfig = ({
   unWrap,
   protocol: type,
 }: CapabilityConfigProps) => {
+  const { adminClient } = useAdminClient();
   const { t } = useTranslation();
   const { control, watch, setValue } = useFormContext<FormFields>();
   const protocol = type || watch("protocol");
   const clientAuthentication = watch("publicClient");
   const authorization = watch("authorizationServicesEnabled");
   const isFeatureEnabled = useIsFeatureEnabled();
-
+  const [idps, setIdps] = useState<IdentityProviderRepresentation[]>([]);
+  const [search, setSearch] = useState("");
+  const { hasSomeAccess } = useAccess();
+  const showIdentityProviders = hasSomeAccess("view-identity-providers");
+  useFetch(
+    async () => {
+      const params: IdentityProvidersQuery = {
+        max: 20,
+        realmOnly: true,
+      };
+      if (search) {
```

Comment:
Note: Ideally we filter this just to identity providers, which are capable to deal with authorization-grants. But that is probably a follow-up... We will probably need some changes on server-side for this as well. WDYT?

If you agree, can you possibly create follow-up issue for this one please?